bootstrap/components/sinks/v002__druid-sink-topology-component.json

```diff
@@ -0,0 +1,352 @@
+{
+  "type": "SINK",
+  "name": "Druid",
+  "subType": "DRUID",
+  "builtin": true,
+  "streamingEngine": "STORM",
+  "fieldHintProviderClass": "com.hortonworks.streamline.streams.cluster.bundle.impl.DruidSinkBundleHintProvider",
+  "transformationClass": "com.hortonworks.streamline.streams.layout.storm.DruidBoltFluxComponent",
+  "mavenDeps": "org.apache.storm:storm-druid:STORM_VERSION,org.scala-lang:scala-library:2.11.8",
+  "topologyComponentUISpecification": {
+    "fields": [
+      {
+        "uiName": "Batch Size",
+        "fieldName": "maxBatchSize",
+        "isOptional": true,
+        "tooltip": "Druid Tranquility: Maximum number of messages to send at once",
+        "type": "number",
+        "defaultValue": 2000
+      },
+      {
+        "uiName": "Max Pending Batches",
+        "fieldName": "maxPendingBatches",
+        "isOptional": true,
+        "tooltip": "Druid Tranquility: Maximum number of batches that may be in flight",
+        "type": "number",
+        "defaultValue": 5
+      },
+      {
+        "uiName": "Linger millis",
+        "fieldName": "lingerMillis",
+        "isOptional": true,
+        "tooltip": "Wait this long for batches to collect more messages (up to maxBatchSize) before sending them.",
+        "type": "number",
+        "defaultValue": 0
+      },
+      {
+        "uiName": "Block On Full",
+        "fieldName": "blockOnFull",
+        "isOptional": true,
+        "tooltip": "Whether send will block (true) or throw an exception (false) when called while the outgoing queue is full",
+        "type": "boolean",
+        "defaultValue": true
+      },
+      {
+        "uiName": "Discarded Events Stream Id",
```

Comment:
Does this eventually translate to driud bolt's output stream? If so, not sure it is really being used given that we dont allow any connections out of a sink in SAM. Also, the edges and connections between components are created by SAM. How does SAM leverage this user input value to name the output stream in flux yaml?

Reply:
Agree. Currently we are just exposing the underlying druid bolt property. This is not used by SAM.  If required will remove this from druid sink.  will do in another PR.